packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -311,74 +286,6 @@ describeCompat("Named root data stores", "FullCompat", (getTestObjectProvider) =
 			assert.ok(await getAliasedDataStoreEntryPoint(dataObject3, alias));
 		});
 
-		it(
-			"Assign multiple data stores to the same alias, first write wins, " +
-				"different containers from snapshot",
-			async function () {
-				// TODO: Re-enable after cross version compat bugs are fixed - ADO:6978
-				if (provider.type === "TestObjectProviderWithVersionedLoad") {
-					this.skip();
-				}
-
-				await setupContainers({
-					...testContainerConfig,
-					runtimeOptions: {
-						summaryOptions: {
-							summaryConfigOverrides: {
-								...DefaultSummaryConfiguration,
-								...{
-									minIdleTime: IdleDetectionTime,
-									maxIdleTime: IdleDetectionTime,
-									maxTime: IdleDetectionTime * 12,
-									initialSummarizerDelayMs: 10,
```

Comment:
These settings aren't impactful to the test logic/result?

Reply:
I shifted the test to a summarizeNow flow.